spring-boot-samples/spring-boot-sample-cache/src/main/resources/infinispan.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan xmlns="urn:infinispan:config:7.2">
 
-	<cache-container default-cache="default">
+	<cache-container default-cache="countries">
```

Comment:
Yeah, but you also need to provide a default cache definition. That file, as it was, would not work because there was no cache named `default` in the file. There were two ways to solve it:

a. Add `default` cache definition, e.g.

```
<local-cache name="default"/>
```

b. Set `countries` as default cache.

I opted for b. since it was less config. It means `countries` cache could be retrieved via `getCache()` or `getCache("countries")` call.